src/StravaClientV2.php

```
@@ -0,0 +1,69 @@
+     * @throws RequestException
+     * @throws JsonException
+     */
+    public function exchangeToken(string $code, string $grant_type = 'authorization_code'): array
+    {
+        $response = $this->strava->exchangeToken($code, $grant_type);
+
+        if ($response->successful()) {
+            return $response->json();
+        }
+
+        throw new \RuntimeException('Failed to exchange token: '.$response->status(), $response->status());
```

PHPMD (2.15.0)

Missing class import via use statement (line '49', column '19'). (Clean Code Rules)

(MissingImport)